force-app/main/default/classes/DummyJSONCallout.cls

```diff
@@ -33,19 +32,54 @@ public with sharing class DummyJSONCallout {
 	 * 
      * @param dummyUserId The ID of the user in the external system to fetch data for.
      */
+	
+	@future(callout=true)
 
 	public static void getDummyJSONUserFromId(String dummyUserId) {
-		// Create HTTP request to send.
 
+		// Create HTTP request to send.
+		Http http = new Http ();
+		HttpRequest req = new HttpRequest();
 		// Set the endpoint URL. Use direct URL or for best practices use Named Credential.
 
-		// Set the HTTP method to GET.
+		req.setEndpoint('https://dummyjson.com/user/' + dummyUserId);
+				// Set the HTTP method to GET.
+
+		req.setMethod('GET');
 
 		// Send the HTTP request and get the response.
+		HttpResponse response = http.send(req);
 
 		// If the HTTP response code is successful, parse the JSON response and update the contact.
-		Contact cont = null;
-		upsert cont DummyJSON_Id__c; //insert/update from the JSON response using the external id (dummyUserId)
+
+		if (response.getStatusCode() == 200) {
+
+			//deserialize into DummyJSONUser
+			DummyJSONUser bogus = (DummyJSONUser)JSON.deserialize(response.getBody(), DummyJSONUser.class);  			//Cast one variable into another
```

Comment:
Be careful to use meaningful variable names. Helps with readability later, `bogus` does not really tell much.